src/utils.tsx

```diff
@@ -402,3 +402,34 @@ export function diffJson(
     return prefixedChunk;
   });
 }
+
+// allows us to share something between components outside of providers
```

Comment:
nit: add note for future selves that we could have used a context provider, but it would have been confusing with redux still in place